packages/client/CHANGELOG.md

```diff
@@ -6,50 +6,50 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+After over a year of continued development the EthereumJS team is proud to announce that
+our client is finally as stable, feature-complete (or - let's call it: feature-consistent)
+and useful in many regards that we have decided to do a first official release! 🌼
 
-**Dependencies, CI and Docs**
+There have been done such an extensive amount of work pieces during that time that it 
+would be neither useful nor enlightening to list all associated PRs.
+
+So here are just a few milestones, where the ones till November 2020 still happened in
+the - now archived - standalone `ethereumjs-client` monorepo before we moved the client
+to our monorepo 
+[ethereumjs-monorepo](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/client):
+
+- Full transition to `TypeScript`, PR [#144](https://github.com/ethereumjs/ethereumjs-client/pull/144) and subsequent PRs
+- Integration with our monorepo CI config, e.g. PR [#153](https://github.com/ethereumjs/ethereumjs-client/pull/153)
+- Full alignment with the latest versions of the `EthereumJS` monorepo libraries (`Block`, `Tx`, `VM`,...), e.g. PR [#158](https://github.com/ethereumjs/ethereumjs-client/pull/158)
+- Integration of the `@ethereumjs/vm` to execute on blocks, PR [#1028](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1028)
+- EIP-1459: DNS peer recovery, PR [#1070](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1070)
+- Following the VM integration: working down various HF bugs like this one from TangerineWhistle (hardening our VM along the way): PR [#1101](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1101)
+- Tip-of-the-chain syncing behavior, PR [#1132](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1132)
+- A basic transaction pool, PR [#1176](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1176)
+- Upgraded `devp2p` `ETH` protocol up to version `ETH/66`, PR [#1331](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1331)
+- `LES` protocol v3 and v5 support, PR [#1324](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1324)
+- Support for `devp2p` snappy compression (RLPx v5), PR [#1399](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1399)
+- A simple miner for PoA/clique, PR [#1444](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1444)
+- Finalized on `devp2p` message handling support (e.g. by adding `NEW_BLOCK` rebroadcast capabilities) to serve as a good network citizen, PR [#1458](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1458)
+- Added developer capabilities, e.g. by adding a `--dev` flag to auto-start and prefund a miner, PR [#1492](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1492)
+
+And here we are. 🙂
+
+We joined various ephemeral testnets along the way - like `Yolov3` (still remember? 😃) or `Calaveras` - and helped find latest consensus bugs before they reach `mainnet` in preparation for the latest forks like `london` in conjunction with the other clients.
```

Comment:
```suggestion
We joined various ephemeral testnets along the way - like `Yolov3` (still remember? 😃) or `Calaveras` - and helped find consensus bugs before they reach `mainnet` in preparation for the latest forks like `london` in conjunction with the other clients.
```

Seems more fluent to me without the duplicate 'latest'

Reply:
```suggestion
We joined various ephemeral testnets along the way - like `Yolov3` (still remember? 😃) and `Calaveras` - and helped find consensus bugs before they reached `mainnet` in preparation for the latest forks like `london` in conjunction with the other clients.
```